openslides/core/finders.py

```diff
@@ -0,0 +1,43 @@
+import os
+from typing import Any, List
+
+from django.conf import settings
+from django.contrib.staticfiles.finders import BaseFinder
+
+from ..core.views import WebclientJavaScriptView
+
+
+class WebclientJavaScriptFinder(BaseFinder):
+    """
+    A dummy finder to create the webclient files when collectstatic is called.
```

Comment:
Hm. This looks like a hack a new developer can hardly understand. Is it possible and easy to override the collect static command instead?

(Also I do no like the magic that running collect static will not only collect some things but also copy things to the directory inside the openslides module.)

Reply:
On the contrary, that is exactly the django way. To overwrite the collect static command is the hack. Then it does not work with the development server, that also uses the finders and not the command.

Reply:
Ohh, I see you already changed the code. I won't veto it, but I really recommend to use the finder. This is the way how different django modules can add files to collect_static.